internal/commands/push/inputs_test.go

```diff
@@ -65,13 +66,13 @@ func TestPushInputsResolve(t *testing.T) {
 		assert.Equal(t, errProjectInvalid{path: localPath, pathExists: true}, i.Resolve(profile, nil))
 	})
 
-	t.Run("Should set the app data if no flags are set but is run from inside a project directory", func(t *testing.T) {
+	t.Run("should set the app data if no flags are set but is run from inside a project directory", func(t *testing.T) {
 		profile, teardown := mock.NewProfileFromTmpDir(t, "app_init_input_test")
 		defer teardown()
 
 		assert.Nil(t, ioutil.WriteFile(
 			filepath.Join(profile.WorkingDirectory, local.FileConfig.String()),
-			[]byte(`{"app_id": "eggcorn-abcde", "name":"eggcorn"}`),
+			[]byte(fmt.Sprintf(`{"config_version":%d,"app_id": "eggcorn-abcde", "name":"eggcorn"}`, realm.DefaultAppConfigVersion)),
```

Comment:
we can be civilized here still :)
```suggestion
			[]byte(fmt.Sprintf(`{"config_version": %d, "app_id": "eggcorn-abcde", "name":"eggcorn"}`, realm.DefaultAppConfigVersion)),
```